otherlibs/unix/unixLabels.mli

```diff
@@ -13,9 +13,27 @@
 (*                                                                        *)
 (**************************************************************************)
 
+(* NOTE:
+   If this file is unixLabels.mli, run tools/unlabel after editing it to
+   generate unix.mli.
+
+   If this file is unix.mli, do not edit it directly -- edit
+   unixLabels.mli instead.
+ *)
+
+(* NOTE:
+   When adding a type to unixLabels.mli, you must also edit
+   tools/unlabel to add the type there.
+  *)
+
 (** Interface to the Unix system.
-   To use as replacement to default {!Unix} module,
-   add [module Unix = UnixLabels] in your implementation.
+
+   To use the labeled version of this module, add [module Unix = UnixLabels]
+   in your implementation.
+
+   Note: all the functions of this module (except {!Unix.error_message} and
```

Comment:
I think (99.99% of) references should omit the prefix (in this case `Unix`) because when one is looking at the doc of the labelled version, clicking on the link will bring you to the page of the unlabelled module; which could be quite annoying.

I guess this will apply to other files as well (but I've only looked at this one).

Reply:
Yes, I think you are right. I will alter them everywhere appropriate. Checking that they work as links in the output of ocamldoc is probably worth doing too, as a cross-check.